config/projects-to-test/openjdk-19-projects-to-test-on.config

```diff
@@ -0,0 +1,8 @@
+# List of GIT repositories to clone / pull for checking with Checkstyle
+# File format: REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]
+# Please note that bash comments works in this file
+#
+# This file has been created to alleviate confusion in the contribution
+# repository's projects-to-test-on.properties file; these projects
+# must be used with special configurations.
+openjdk19|git|https://github.com/openjdk/jdk19u.git|master||
```

Comment:
I thought it would be better just to create a new file instead of `sed` ing the existing one, so that we can keep the scripts consistent and make removal of the older one easier.